docs/reference/spec/host-capabilities/06-kubernetes.md

```diff
@@ -122,3 +138,83 @@ Use the `list_resources_all` when dealing with cluster-wide resources.
 #### Output
 
 Result of `GET /apis/$api_version/namespaces/$namespace/$kind/$name`
+
+### Operation - `can_i`
+
+This operation determines if a specific user or group has the permission to perform an
+action on a Kubernetes resource. It does this by making a
+[`SubjectAccessReview`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#subjectaccessreview-v1-authorization-k8s-io)
+request to the Kubernetes API server.
+
+#### Input
+
+The `can_i` operation requires a JSON object with the following parameters:
+
+```hcl
+{
+  "subject_access_review": {
+    "groups": null,
+    "resource_attributes": {
+      "group": "",
+      "name": null,
+      "namespace": "kube-system",
+      "resource": "pods",
+      "subresource": null,
+      "verb": "create",
+      "version": null
+    },
+    "user": "system:serviceaccount:customer-1:testing"
+  },
+  "disable_cache": false
+}
+```
+
+#### Output
+
+The output is a JSON object that contains the status of the
+`SubjectAccessReview` request. This object indicates whether the requested
+action is allowed or not. For a complete reference of all the fields available in the
+output, see the official
+[`SubjectAccessReviewStatus`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.32/#subjectaccessreviewstatus-v1-authorization-k8s-io)
+documentation.
+
+:::caution
+It's important to remember that all of the returned values came from the
+Kubernetes authorization API. Therefore, any configuration differences or
+future change in the API can affect the results here as well.
+:::
+
+Example of an allowed operation. Some of the authorization plugins allows the operation:
+
+```hcl
+{
+  "allowed": true, // Allowed is required. True if the action would be allowed, false otherwise.
+  "denied": false, // Denied is optional. True if the action would be denied, otherwise false. If both allowed is false and denied is false, then the authorizer has no opinion on whether to authorize the action. Denied may not be true if Allowed is true.
+  "evaluationError": "EvaluationError is an indication that some error occurred during the authorization check. It is entirely possible to get an error and be able to continue determine authorization status in spite of it. For instance, RBAC can be missing a role, but enough roles are still present and bound to reason about the request.",
+  "reason": "Reason is optional. It indicates why a request was allowed or denied."
```

Comment:
I would prefer all fields marked with a comment for consistency ( `// Reason is optional`).